addons/vsphere-csi-driver/vsphere-csi-driver.yaml

```diff
@@ -7,9 +6,9 @@ metadata:
     kubeaddons.mesosphere.io/name: vsphere-csi-driver
     kubeaddons.mesosphere.io/provides: csi-driver
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.2.1-1"
-    appversion.kubeaddons.mesosphere.io/vsphere-csi-driver: "2.0.1"
-    values.chart.helm.kubeaddons.mesosphere.io/vsphere-csi-driver: "https://raw.githubusercontent.com/mesosphere/charts/5565912/stable/vsphere-csi-driver/values.yaml"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "2.2.0-1"
```

Comment:
Is this the right version?

Reply:
Oh good catch shouldn't this come from https://github.com/mesosphere/charts/blob/a41acbd59fdb11635e8f8119f33308d17045cbde/stable/vsphere-csi-driver/Chart.yaml#L7?